global-script.lua

```diff
@@ -34,6 +34,14 @@ elementScanZones = {
     ["Green"] = "190f05",
     ["Orange"] = "61ac7c",
 }
+playerTables = {
```

Comment:
this should probably appear in the saved config data block?

Reply:
This is no longer necessary.  The old version of this swapped the tints of the player tables, but the new version downright swaps their locations so the mappings of the table no longer change.

Reply:
can you remove playerTables from the save state then?